addons/MountMuzzle/mountmuzzle.lua

```diff
@@ -0,0 +1,222 @@
+
+local settings = _config.load(defaults)
+
+local help = {
+    commands = {
+        buildHelpSeperator('=', 26),
+        buildHelpTitle('Commands'),
+        buildHelpSeperator('=', 26),
+        buildHelpCommandEntry('list', 'Display the available muzzle types.'),
+        buildHelpCommandEntry('set <muzzle>', 'Set the current muzzle to the given muzzle type.'),
+        buildHelpCommandEntry('get', 'Display the current muzzle.'),
+        buildHelpCommandEntry('default', 'Set the current muzzle to the default (Silent).'),
+        buildHelpCommandEntry('reload', 'Reload Mount Muzzle.'),
+        buildHelpCommandEntry('help', 'Display Mount Muzzle commands.'),
+        buildHelpSeperator('=', 26),
+    },
+
+    types = {
+        buildHelpSeperator('=', 23),
+        buildHelpTitle('Types'),
+        buildHelpSeperator('=', 23),
+        buildHelpTypeEntry(ucFirst(muzzles.silent.name), muzzles.silent.description),
+        buildHelpTypeEntry(ucFirst(muzzles.normal.name), muzzles.normal.description),
+        buildHelpTypeEntry(ucFirst(muzzles.choco.name), muzzles.choco.description),
+        buildHelpTypeEntry(ucFirst(muzzles.zone.name), muzzles.zone.description),
+        buildHelpSeperator('=', 23),
+    },
+}
+
+function display_help(table_help)
+    for index, command in pairs(table_help) do
+        displayResponse(command)
+    end
+end
+
+function getMuzzle()
+    return settings.muzzle
+end
+
+function setMuzzle(muzzle)
+    settings.muzzle = muzzle
+    settings:save()
+end
+
+function muzzleValid(muzzle)
+    return muzzles[muzzle] ~= nil
+end
+
+function resolveCurrentMuzzle()
+    local current_muzzle = getMuzzle()
+    if not muzzleValid(current_muzzle) then
+        current_muzzle = muzzles.silent.name
+        setMuzzle(current_muzzle)
+        displayResponse(
+            'Note: Muzzle found in settings was not valid and is now set to the default (%s).':format('Silent':color(colors.secondary)),
+            colors.warn
+        )
+    end
+    return muzzles[current_muzzle]
+end
+
+function requestInject()
+    needs_inject = true
+end
+
+function injectMuzzleMusic()
+    windower.packets.inject_incoming(
+        packets.inbound.music_change.id,
+        'IHH':pack(packets.inbound.music_change.id,
+            music.types.mount,
+            resolveCurrentMuzzle().song
+        )
+    )
+end
+
+function handleInjectionNeeds() 
+    if needs_inject and playerIsMounted() then
+        injectMuzzleMusic()
+        needs_inject = false;
+    end
+end
+
+function playerIsMounted()
+    local _player = windower.ffxi.get_player()
+    if _player then
+        return mounted or _player.status == player.statuses.mounted
+    end
+    return false 
+end
+
+windower.register_event('login', requestInject)
+windower.register_event('load', requestInject)
+windower.register_event('zone change', requestInject)
+
+windower.register_event('addon command', function(command, ...)
+    if command then
+        local command = command:lower()
```

Comment:
You probably want to remove the `local`. Otherwise this statement does nothing. It creates a local variable `command` which *hides* the original `command` variable, and which is immediately discarded, as the inner `command` dies at the `else` keyword.

Reply:
Done.